models/node_model.go

```diff
@@ -34,6 +34,7 @@ const (
 	GenericNodeType   NodeType = "generic"
 	ContainerNodeType NodeType = "container"
 	RemoteNodeType    NodeType = "remote"
+	RDSNodeType       NodeType = "rds"
```

Comment:
```suggestion
	RemoteRDSNodeType       NodeType = "remote-rds"
```

But that also does not belong to this PR.

Reply:
It is. 
![image](https://user-images.githubusercontent.com/11512/69343393-9eed8400-0c7e-11ea-8422-de04b262829d.png) That was before I started fixing it.